assets/cerebras.yaml

```diff
@@ -0,0 +1,76 @@
+---
+
+- type: model
+  name: Cerebras-GPT
+  # General
+  organization: Cerebras
+  description: >
+    A Family of Open, Compute-efficient, Large Language Models. The family includes
+    111M, 256M, 590M, 1.3B, 2.7B, 6.7B, and 13B models. All models in the Cerebras-GPT
+    family have been trained in accordance with Chinchilla scaling laws (20 tokens
+    per model parameter).
+    [[Cerebras Blog Post]](https://www.cerebras.net/blog/cerebras-gpt-a-family-of-open-compute-efficient-large-language-models).
+  created_date:
+    value: 2023-03-28
+    explanation: >
+      The date of the blog post announcing Cerebras-GPT
+      [[Cerebras Blog Post]]
+      (https://www.cerebras.net/blog/cerebras-gpt-a-family-of-open-compute-efficient-large-language-models).
+  url: https://www.cerebras.net/blog/cerebras-gpt-a-family-of-open-compute-efficient-large-language-models/
+  model_card: https://huggingface.co/cerebras/Cerebras-GPT-13B
+  modality: text (English)
+  size: 13B parameters (dense model)
```

Comment:
Could you include in the explanation that this is the size of the largest model?